### PR TITLE
Fix text, result of switching level 5 and 6

### PR DESCRIPTION
**Pull request by OnnoEbbens**

### CODE REVIEW

File: coursedata/level-defaults/nl.yaml [hunk 1]
```diff
@@ -171,7 +171,7 @@
 5:
     start_code: "print '5 keer 5 is ' 5 * 5"
     intro_text: |
-        `ask`, `print`, `if` en `repeat` werken nog zoals in level 4.
+        `ask`, `print` en `if` werken nog zoals in level 4.
```
Comment (Felienne) [inline coursedata/level-defaults/nl.yaml:174]: Nice catch! This same error occurs in all others langs too :( Could you fix these too, or f that is too much work, make an issue so I don't forgot to fix it?

**Review by Felienne — APPROVED** [review]

**Comment by OnnoEbbens** [comment]
I added fixes for the other languages but right before I pushed these you merged the other changes into main. I am not sure which changes are added now. Next time I will reply first and then start doing stuff :)

**Comment by Felienne** [comment]
Ow, sorry!! I merged before so I do not have the changes (I made an issue
because I figured you'd want to have this ready tomorrow)

Can you make a new PR based on your latest changes?

**Comment by OnnoEbbens** [comment]
No worries at all. I made a new PR, I think that will do :)

**Pull request merged**